sass/_FieldTable.scss

```diff
@@ -65,13 +66,18 @@
 
 .coveo-field-table-toggle-icon-up {
   position: absolute;
-  bottom: 10px;
+  bottom: 35px;
   right: 10px;
   margin: 0;
+  @include transition(opacity 0s);
```

Comment:
Does this really need to be specified twice? It's also in the `&.coveo-opened` selector